packages/realm-react/src/index.tsx

```diff
@@ -93,6 +93,7 @@ type RealmContext = {
    * ```
    * @param type - The object type, depicted by a string or a class extending {@link Realm.Object}
    * @param primaryKey - The primary key of the desired object which will be retrieved using {@link Realm.objectForPrimaryKey}
+   * @param keyPaths - Indicates a lower bound on the changes relevant for the hook. This is a lower bound, since if multiple hooks add listeners (each with their own `keyPaths`) the union of these key-paths will determine the changes that are considered relevant for all listeners registered on the object. In other words: A listener might fire and cause a re-render more than the key-paths specify, if other listeners with different key-paths are present.
```

Comment:
I've added this here, but I still need to figure out a way to document the overload on `useQuery`.

Reply:
Should we deprecate the previous overloads?  Then you can just document the new way and mention the old as deprecated.